docs/images-location.md

```diff
@@ -0,0 +1,362 @@
+      </td>
+      <td>
+        <a href="https://github.com/kubeflow/katib/blob/master/cmd/earlystopping/medianstop/v1beta1/Dockerfile">Dockerfile</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Training Containers Images
+
+The following table shows images for training containers which are used in the
+[Katib Trials](https://www.kubeflow.org/docs/components/katib/experiment/#packaging-your-training-code-in-a-container-image).
+
+<table>
+  <tbody>
+    <tr align="center">
+      <td>
+        <b>Image Name</b>
+      </td>
+      <td>
+        <b>Description</b>
+      </td>
+      <td>
+        <b>Location</b>
+      </td>
+    </tr>
+    <tr align="center">
+      <td>
+        <code>docker.io/kubeflowkatib/mxnet-mnist</code>
+      </td>
+      <td>
+        MXNet MNIST example with collecting metrics time
+      </td>
+      <td>
+        <a href="https://github.com/kubeflow/katib/blob/master/examples/v1beta1/trial-training-containers/mxnet-mnist/Dockerfile">Dockerfile</a>
+      </td>
+    </tr>
+    <tr align="center">
+      <td>
+        <code>docker.io/kubeflowkatib/pytorch-mnist</code>
+      </td>
+      <td>
+        PyTorch MNIST example with printing metrics to the file or StdOut
+      </td>
+      <td>
+        <a href="https://github.com/kubeflow/katib/blob/master/examples/v1beta1/trial-training-containers/pytorch-mnist/Dockerfile">Dockerfile</a>
+      </td>
+    </tr>
+    <tr align="center">
+      <td>
+        <code>gcr.io/kubeflow-ci/tf-mnist-with-summaries</code>
+      </td>
+      <td>
+        Tensorflow MNIST example with saving metrics in the summaries
+      </td>
+      <td>
+        <a href="https://github.com/kubeflow/tf-operator/blob/2712f5667ec78f17d22288630f8719f0c08990ba/examples/tensorflow/mnist_with_summaries/Dockerfile">Dockerfile</a>
+      </td>
+    </tr>
+    <tr align="center">
+      <td>
+        <code>docker.io/kubeflowkatib/xgboost-lightgbm</code>
+      </td>
+      <td>
+        Distributed LightGBM example for XGBoostJob
+      </td>
+      <td>
+        <a href="https://github.com/kubeflow/xgboost-operator/blob/9c8c97d0125a8156f12b8ef5b93f99e709fb57ea/config/samples/lightgbm-dist/Dockerfile">Dockerfile</a>
+      </td>
+    </tr>
+    <tr align="center">
+      <td>
+        <code>docker.io/kubeflow/mpi-horovod-mnist</code>
+      </td>
+      <td>
+        Distributed Horovod example for MPIJob
+      </td>
+      <td>
+        <a href="https://github.com/kubeflow/mpi-operator/blob/947d396a9caf70d3c94bf587d5e5da32b70f0f53/examples/horovod/Dockerfile.cpu">Dockerfile</a>
+      </td>
+    </tr>
+    <tr align="center">
+      <td>
+        <code>docker.io/inaccel/jupyter:lab</code>
+      </td>
+      <td>
+        FPGA XGBoost with parameter tuning
+      </td>
+      <td>
+        <a href="https://github.com/inaccel/jupyter/blob/8939209cc7958d07947055945dc99e0ce3c3c4ba/lab/Dockerfile">Dockerfile</a>
```

Comment:
```suggestion
        <a href="https://github.com/inaccel/jupyter/blob/master/lab/Dockerfile">Dockerfile</a>
```

We would prefer not to point to a specific commit, `master` branch will always describe the latest stable image (`docker.io/inaccel/jupyter:lab`).

Reply:
Sure, I will change it.
@eliaskoromilas You are not planing to change directory, right ?

Reply:
Right. It matches the image the tag.